userspace/falco/app_actions/print_ignored_events.cpp

```diff
@@ -28,7 +28,7 @@ void application::print_all_ignored_events()
 	std::set<string> ignored_event_names;
 	for(uint32_t j = 0; j < PPM_EVENT_MAX; j++)
 	{
-		if(!sinsp::simple_consumer_consider_evtnum(j))
+		if(!simple_consumer_consider(etable[j].flags))
```

Comment:
Probably also here we need to find a way to check which events are in the Falco interesting set, the only problem here is that we don't have the Falco set populated at this point in time